projects/go-style-guide/src/app/features/ui-kit/components/off-canvas-docs/off-canvas-docs.component.ts

```diff
@@ -58,9 +58,11 @@ export class OffCanvasDocsComponent {
     this.goOffCanvasService.openOffCanvas({
       component: BasicTestComponent,
       bindings: {
-        someBinding: 'monkey'
```

Comment:
This is an example of how you could pass anything into the Components/Off-Canvas. This property didn't exist anywhere on the Off Canvas or BasicTestComponent.